File: client/public/Codes/C/Swap.c
#include <stdio.h>

void swap(int a, int b)
{
    a = a + b;
    b = a - b;
    a = a - b;
    printf("After Swapping:\n");
    printf("a = %d, b = %d", a, b);
}

int main()
{
    int a, b;
    printf("Enter 2 Numbers:");
    scanf("%d %d", &a, &b);
    printf("Before Swapping:\n");
    printf("a = %d, b = %d\n", a, b);
    swap(a, b);
    return 0;
}

File: client/public/Codes/Java/Swap.java

import java.util.Scanner;

public class Swap {
    public static void swap(int a, int b) {
        a = a + b;
        b = a - b;
        a = a - b;
        System.out.println("After Swapping:");
        System.out.println("a = " + a + ", b = " + b);
    }

    public static void main(String[] args) {
        int a, b;
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter 2 Numbers:");
        a = scanner.nextInt();
        b = scanner.nextInt();
        System.out.println("Before Swapping:");
        System.out.println("a = " + a + ", b = " + b);
        swap(a, b);

        scanner.close();
    }
}
